archlinux/PKGBUILD

```diff
@@ -21,21 +20,39 @@ backup=()
 options=()
 changelog=
 
-source=(PKGBUILD-z-qubes-session.sh)
+source=()
+md5sums=()
 
 noextract=()
-md5sums=() #generate with 'makepkg -g'
-pa_ver=$((pkg-config --modversion libpulse 2>/dev/null || echo 0.0) | cut -f 1 -d "-")
+pa_ver=16.1 #specify this manually please
+# If using pkg-config, you must update libpulse first.
```

Comment:
Automatic builds (via qubes-builder-archlinux) _do_ install updates before building a package, so I'd prefer to keep it automatic (one place less to change by hand).